examples/transformerconfigs/crd/README.md

```diff
@@ -0,0 +1,133 @@
+## Transformer Configurations - CRD
+
+This tutorial shows how to add transformer configurations to support a CRD type.
+
+### Get Default Config
+Get the default transformer configurations by
+
+<!-- @saveConfig @test -->
+```
+kustomize config save -d ~/.kustomize/config
+```
+The default configurations are save in directory `~/.kustomize/config` as several files
+
+> ```
+>  commonannotations.yaml  commonlabels.yaml  nameprefix.yaml  namereference.yaml  namespace.yaml  varreference.yaml
+> ```
+
+### Add Config for a CRD
+All transformers will be involved for a CRD type. The default configurations already include some common fieldSpec for all types:
+
+- nameprefix is added to `.metadata.name`
+- namespace is added to `.metadata.namespace`
+- labels is added to `.metadata.labels`
+- annotations is added to `.metadata.annotations`
+
+Thus those fieldSpec don't need to be added to support a CRD type.
+Consider a CRD type `MyKind` with fields
+- `.spec.secretRef.name` reference a Secret
+- `.spec.beeRef.name` reference an instance of CRD `Bee`
+- `.spec.containers.commands` as the list of container commands
+- `.spec.selectors` as the label selectors
+
+Add following file to configure the transformers for the above fields
+<!-- @addConfig @test -->
+```
+cat > ~/.kustomize/config/mykind.yaml << EOF
+
+commonLabels:
```

Comment:
this `commonLabels` transformer config is not demonstrated in the example below. I guess you need to add a `commonLabels` config in the example kustomization.yaml? Not sure if this is intentional or not but I was expecting to see something happen to the output based on this transformer when I was running through the tutorial.

(same with the varReference transformer config below).

Reply:
good point. Let me add those fields in the demo.